mcookie: add page (#3322)
tldr-pages/tldr: pages/linux/mcookie.md

# mcookie

> > Generate a 128-bit random hexadecimal number.

Reviewer: Perhaps this will read better?
```suggestion
> Generates random 128 bit numbers in hexadecimal.
```

Author: @sbrl What about this ?

Generates random 128 bit hexadecimal number.

Reviewer: Almost, @nsuthison. You're missing an article there.
```suggestion
> Generates random 128 bit hexadecimal numbers.
```
It either has to be plural (`numbers`), or include an article (`a random number`).

> - Generate random number:

Reviewer: Perhaps this will read better?
```suggestion
- Generate a random number:
```

`mcookie`

> - Generate random number, use destination file as a seed for the randomness:

Reviewer: Perhaps this will read better?
```suggestion
- Generate a random number, using the contents of a file as a seed for the randomness:
```

Also, are there long forms of these options? In tldr-pages we prefer to use long forms of arguments to improve clarity.

Author: > In tldr-pages we prefer to use long forms of arguments to improve clarity.

Ok, I'll change it right away.

> - Generate a random number, using the contents of a file as a seed for the randomness:

Reviewer 2: I think we can just use one file in these examples, to keep it simple.

Author: I quite not sure about this. Let us have the third opinion please. @sbrl

Reviewer 3: Well having 2 files surely explains better what the command does. But the whole idea behind tldr is to exemplify in the most simple way the most common form of use of the commands we maintain.

I would prefer to have only one file on the example as more than one could cause some confusion.

Author: Ok, I'll change it to have just only one file. Thank you for your comments! @agnivade @schneiderl

Reviewer: Hmmmm. I think, in this instance, if someone wants to specify multiple files, it should be fairly intuitive to use `--file` multiple times. To this end, I think it's ok if we don't show this off specifically here.

It sounds like an advanced use-case, which generally isn't a good fit for a tldr page. If it's something that users of this command would need regularly, then we should probably reconsider.

`mcookie -f {{path/to/file1}} -f {{path/to/file2}}`

> - Generate random number, read specific number of bytes from destination file and use it as a seed for the randomness:

Reviewer: Perhaps this will read better?
```suggestion
- Generate a random number, using a specific number of bytes from a file as a seed for the randomness:
```

`mcookie -f {{path/to/file1}} -m {{number_of_bytes}} -f {{path/to/file2}} -m {{number_of_bytes}}`

> - Print the detials of the randomness such as origin and seed from each sources:

Reviewer: Perhaps this will read better?
```suggestion
- Print the details of the randomness used, such as the origin and seed for each source of randomness:
```

Author: @sbrl What about this?

- Print the details of the randomness used, such as the origin and seed for each source:

Reviewer: Sounds good to me!

Author: Thank you a lot! :D
